Taskfile.yaml

```
@@ -74,4 +74,33 @@ tasks:
     deps:
       - build
     cmds:
-      - docker build --build-arg VERSION={{.VERSION}} --build-arg SHA=$(SHA) -t {{.REGISTRY}}/{{.COMPONENT}}:latest .
\ No newline at end of file
+      - docker build --build-arg VERSION={{.VERSION}} --build-arg SHA=$(SHA) -t {{.REGISTRY}}/{{.COMPONENT}}:latest .
+
+  start:
+    desc: start the test SAP Hana database
+    dir: tests
+    cmds:
+      - docker-compose up -d
+  
+  stop:
+    desc: stop the test SAP Hana database
+    summary: |
+      This task stops the test SAP Hana database.
+
+      Examples:
+        task stop
+        task stop -- -v
+    dir: tests
+    cmds:
+      - docker-compose down {{.CLI_ARGS}}
+
+  ingest:
+  cmds:
+    - poetry run datahub ingest -c datahub_sap_hana/recipe.yaml
```

is the recipe nested inside the python module? i think it would be better for those recipes to live in the project root, so it would just be 
```suggestion
    - poetry run datahub ingest -c recipe.yaml
```